.changeset/strong-numbers-warn.md

```diff
@@ -0,0 +1,5 @@
+---
+'@directus/errors': major
```

Comment:
Not a breaking change, IMO
```suggestion
'@directus/errors': minor
```

Reply:
Oh yes. I was under the impression that `reason` is a required argument now. Minor it is.